src/login/logind-core.c

```diff
@@ -290,6 +290,8 @@ int manager_get_session_by_pid(Manager *m, pid_t pid, Session **session) {
         if (pid < 1)
                 return -EINVAL;
 
+        /* Get client session.  This will not work consistently because many processes may be started
```

Comment:
Again, "consistently" is not the right adjective, because it suggests that the results are flaky. But that's not the case, they are reliable, just many processes are not part of a seesion. What about replacing the adjective with "in all cases" and dropping the last sentence, since the API was and still is defined in this way.